src/Foundation/Http/Middleware/CheckForMaintenanceMode.php

```diff
@@ -23,26 +23,25 @@ public function handle($request, Closure $next)
     {
         try {
             return parent::handle($request, $next);
-        } catch (\Illuminate\Foundation\Http\Exceptions\MaintenanceModeException $ex) {
-            // Smoothly handle AJAX requests
-            if (request()->ajax()) {
-                return Response::make(Lang::get('system::lang.page.maintenance.help') . "\r\n" . $ex->getMessage(), 503);
-            }
+        } catch (\Symfony\Component\HttpKernel\Exception\HttpException $ex) {
+            if ($ex->getStatusCode() === 503) {
+                // Smoothly handle AJAX requests
+                if (request()->ajax()) {
+                    return Response::make(Lang::get('system::lang.page.maintenance.help') . "\r\n" . $ex->getMessage(), 503);
+                }
 
-            // Check if there is a project level view to override the system one
-            View::addNamespace('base', base_path());
-            if (View::exists('base::maintenance')) {
-                $view = 'base::maintenance';
-            } else {
-                $view = 'system::maintenance';
-            }
+                // Check if there is a project level view to override the system one
+                View::addNamespace('base', base_path());
+                if (View::exists('base::maintenance')) {
+                    $view = 'base::maintenance';
+                } else {
+                    $view = 'system::maintenance';
+                }
 
-            return Response::make(View::make($view, [
-                'message'           => $ex->getMessage(),
-                'wentDownAt'        => $ex->wentDownAt,
-                'retryAfter'        => $ex->retryAfter,
-                'willBeAvailableAt' => $ex->willBeAvailableAt,
-            ]), 503);
+                return Response::make(View::make($view, [
+                    'message'           => $ex->getMessage()
+                ]), 503);
```

Comment:
Any concerns with these variables no longer being provided?

Reply:
I didn't make that change, but those properties no longer exist in Symfony's HttpException class.